### ISSUE TITLE
add bookmarks to PDF exported file

### ISSUE THREAD

**Issue by Erwan-le-Gall** [suitenumerique/docs#1135]
label: accessibility

When I create a Docs document with headers

Then the resulting exported PDF document must have an outline of bookmarks based on these headers

reference: https://opensource.adobe.com/dc-acrobat-sdk-docs/pdfstandards/PDF32000_2008.pdf#page=375

This is an accessibility requirement for WCAG and RGAA compliance.

**Comment by Erwan-le-Gall** [comment]
ping @cyberbaloo

**Comment by virgile-dev** [comment]
Hey @Erwan-le-Gall 
Thanks for opening all those accessibility related issue?
Do you think you could contribute and work on fixing a few? 
@AntoLC should be able to review them during the summer.

**Comment by Erwan-le-Gall** [comment]
I'm no more a dev… :(

_But_ I think the answer is in [React-PDF's documentation](https://react-pdf.org/components#document): https://react-pdf.org/components#document :

- bookmark	Attach bookmark to element
- useOutlines	Document bookmarks visible

There's an example code in https://react-pdf.org/advanced#bookmarks

Docs should have such bookmark attributes defined on headers.

ping @AntoLC we can have a talk about it and I can test a generated PDF and give you feedback

**Comment by Erwan-le-Gall** [comment]
thanks Cyril @Ovgodd :)
